Sources/RSDKUtilsMain/AnalyticsBroadcaster.swift

```diff
@@ -2,8 +2,8 @@ import Foundation
 
 // MARK: - Constants
 
-extension NSNotification.Name {
-    static let sdkCustomEvent = Notification.Name(rawValue: "com.rakuten.esd.sdk.events.custom")
+public extension NSNotification.Name {
+    static let rAnalyticsCustomEvent = Notification.Name(rawValue: "com.rakuten.esd.sdk.events.custom")
```

Comment:
I'm open to other ideas for the name of this constant

Reply:
I don't have better than this name.

Reply:
I want to replace `esd` with `tech` so it's `com.rakuten.tech.sdk.events.custom`